assets/javascripts/discourse/widgets/event-invitation-notification-item.js

```diff
@@ -4,6 +4,9 @@ import { DefaultNotificationItem } from "discourse/widgets/default-notification-
 import { escapeExpression, formatUsername } from "discourse/lib/utilities";
 import { iconNode } from "discourse-common/lib/icon-library";
 
+// TODO: delete the strings marked with TODO in translation files when
```

Comment:
This is nothing to do with this PR specifically, but I often find it easier to find my own TODOs in the future if i add the suffix (name) e.g. `# TODO (martin)`. Don't have to follow this just thought I would note it :)

Reply:
done!